inference_cli/main.py

```diff
@@ -14,6 +14,26 @@
 app.add_typer(benchmark_app, name="benchmark")
 
 
+def version_callback(value: bool):
+    if value:
+        from importlib.metadata import version, PackageNotFoundError
+
+        try:
+            inference_version = version('inference')
```

Comment:
We also have `inference_sdk` which would be nice to be grabbed and echoed back to the user.  Displaying version of `inference_cli` itself would be useful too.